### ISSUE TITLE
I Request, if more utils can be integrated.

### ISSUE THREAD

**Issue by akczht** [holzschu/a-shell#196]
I Request to the dev to include mote utils like youtube-dl , spotdl , ffmpeg and transmission to download youtube videos , spotify songs and torrents , I know it's out of the app's intended use but please these utils won't disturb anything, youtube-dl can still be downloaded via pip , but natively supporting it would be better and spot-dl doesn't work , I request to integrate that too and I don't know about transmission but , it might be against apple's appstore policies to integrate that but allowing us to install that would be so much helpful and ffmpeg is just good .

**Comment by akczht** [comment]
I think iSH is simply better for my use case , thanks guys .

**Comment by akczht** [comment]
> I Request to the dev to include more utils like youtube-dl , spotdl , ffmpeg and transmission to download youtube videos , spotify songs and torrents , I know it's out of the app's intended use but please these utils won't disturb anything, youtube-dl can still be downloaded via pip , but natively supporting it would be better and spot-dl doesn't work , I request to integrate that too and I don't know about transmission but , it might be against apple's appstore policies to integrate that but allowing us to install that would be so much helpful and ffmpeg is just good .

@holzschu

**Comment by holzschu** [comment]
Hi, 
thanks for the suggestions. I'll see what I can do. The key idea is to let users customise the app to satisfy their specific needs, but if something doesn't work, like spot-dl, that's another story. 

Transmission looks extremely difficult with the current limitations of the architecture (not just policy).

**Comment by Akczht** [comment]
> Hi, 
> 
> thanks for the suggestions. I'll see what I can do. The key idea is to let users customise the app to satisfy their specific needs, but if something doesn't work, like spot-dl, that's another story. 
> 
> 
> 
> Transmission looks extremely difficult with the current limitations of the architecture (not just policy). 

Transmission is not necessary, but here is an issue , the text from last sessions remains even after closing the app . Ffmpeg spot-dl (if possible) and youtube-dl are my request to be integrated in the app's code itself. And I think vtt is more clunky and sluggish , why not approach it in a way , termite did on linux , by not using vtt and using something from termite's source code .

Thanks!

**Comment by holzschu** [comment]
> the text from last sessions remains even after closing the app 

Yes, that is part of Apple's user experience requirements: when you switch to another app and then come back to this app, it should behave as if it had not terminated: same screen, same command history, etc. 

If you type the command "exit", then the screen and command history will be cleared.
